paddle/framework/attr_test.proto

```diff
@@ -0,0 +1,8 @@
+syntax="proto3";
```

Comment:
I understand unit test source files are named something like `xxx_test.cc`, but what is `xxx_test.proto` for?